SpatialConnectTests/tests.scfg

```diff
@@ -27,6 +27,13 @@
 "name":"Whitehorse",
 "uri":"https://portal.opengeospatial.org/files/63156",
 "id":"ba293796-5026-46f7-a2ff-e5dec85heh6b"
+},
+{
+"store_type":"wfs",
+"version":"1",
```

Comment:
@marcusthebrown it is the version of the store. It needs to be longer term a float instead of an int.